constraints/strongStart.js

```diff
@@ -214,3 +291,32 @@ function strongStartInit(stree, ptree, cat){
 }
 
 
+//Searches ptree for node of cat p, then checks to see if it dominates (or is itself) the parent node (id) of the node of cat k under question.
+function seekCatLeftEdge(node, p, id){
+	if(!node.children){
+		return 0;
+	}
+	if(node.cat === p && (searchDownLeft(node, id) || node.id === id)){
+		return 1;
+	}
+	else {
+		for(var i=0; i<node.children.length; i++){
+			if(seekCatLeftEdge(node.children[i], p, id)){
+				return 1;
+			}
+		}
+	}
+}
+
+//Recurses down the first children a node to check whether or not it dominates a node with the specified id at its left edge.
```

Comment:
Improve this comment as well. "A helper function for [function name]. Takes [argument list and info]. Returns true if [whatever conditions]. Returns false otherwise."

Please change the return values to true/false instead of using 1/0.

Reply:
I think this function name could be more transparent as well. hasIdAtLeftEdge(node, id)?